keep/api/core/db.py

```diff
@@ -3175,12 +3175,10 @@ def get_incident_by_id(
 
 
 def create_incident_from_dto(
-    tenant_id: str, incident_dto: IncidentDtoIn | IncidentDto
+    tenant_id: str, incident_dto: IncidentDtoIn | IncidentDto, generated_from_ai: bool = False
 ) -> Optional[Incident]:
-    # from AI
-    if isinstance(incident_dto, IncidentDto):
-        # get all the fields from the DTO
 
+    if issubclass(type(incident_dto), IncidentDto) and generated_from_ai:
```

Comment:
This is a bit "tricky", what if someday we'll have a generated_from_ai incident coming from a 3rd party provider? isn't there a "clearer" way to do this?